### PR TITLE
mesa: separate out libgbm-devel.

### PR DESCRIPTION
**Pull request by mahiuchun**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [X] I generally don't use the affected packages but briefly tested this PR

Since there is libgbm, there should be libgbm-devel as well.

### CODE REVIEW

File: srcpkgs/mesa/template [hunk 1]
```diff
@@ -228,6 +228,16 @@ libgbm_package() {
 	}
 }
 
+libgbm-devel_package() {
+	short_desc="Mesa Generic buffer management API - development files"
+	depends="libgbm>=${version}_${revision}"
+	pkg_install() {
+		vmove /usr/include/gbm.h
+		vmove /usr/lib/libgbm.so
+		vmove /usr/lib/pkgconfig/gbm.pc
```
Comment (ericonr) [inline srcpkgs/mesa/template:237]: Shouldn't use a leading slash in paths for `vmove`.

**Review by ericonr — COMMENTED** [review]
Is there a direct reason for it? Does it make a difference for any packages we build?

**Comment by st3r4g** [comment]
> Is there a direct reason for it? Does it make a difference for any packages we build?

I think it's good to optimize the build time of some packages, such as wayland compositors. The `MesaLib-devel` makedep could be reduced to`libglvnd-devel` + `libgbm-devel`. See https://github.com/void-linux/void-packages/issues/23243

File: srcpkgs/mesa/template (hunk 1 above)
Comment (mahiuchun) [reply srcpkgs/mesa/template]: done

**Comment by ericonr** [comment]
I think it'd be nice to include such improvements here already, otherwise people are unlikely to proactively discover them when touching the affected packages (note that they probably won't require revbumps).

**Comment by mahiuchun** [comment]
I did it for `wlroots`.

**Pull request closed without merging**